kubernetes/.swagger-codegen-ignore

```diff
@@ -21,3 +21,6 @@
 #docs/*.md
 # Then explicitly reverse the ignore rule for a single file:
 #!docs/README.md
+
+lib/Kubernetes/Watch/*.hs
```

Comment:
I just noticed this doesn't actually ignore `hs` files in this directory when running the code generator. Will look into it.